tests/bootstrap.php

```diff
@@ -47,5 +47,6 @@ class_alias("Contao\\$class", $class);
 
 /** @var Composer\Autoload\ClassLoader $loader */
 $loader->addPsr4('Contao\\CoreBundle\\Test\\', __DIR__);
+$loader->addPsr4('Contao\\Fixtures\\', __DIR__.'/Fixtures');
```

Comment:
@discordier is absolutely right. That's a bit unorthodox in SF2's world. So I set this namespace as usual.